spec/tasks/summaries_spec.rb

```diff
@@ -0,0 +1,70 @@
+require "rails_helper"
+require "rake"
+
+describe "summaries.rake rake tasks" do
+  let(:task) { Rake::Task[task_name] }
+  let(:task_name) { "my:rake:task" }
+
+  after(:each) do
+    task.reenable
+  end
+
+  describe "daily" do
+    let(:task_name) { "summaries:daily" }
+
+    it "sends a daily price summary" do
+      expect(HappyHood::Slack::Client).to receive(:send_daily_price_summary)
+
+      task.invoke
+    end
+
+    context "with caching enabled" do
+      before do
+        file_store_cache = ActiveSupport::Cache.lookup_store(:file_store, "tmp/test#{ENV["TEST_ENV_NUMBER"]}/cache")
```

Comment:
Where's `ENV['TEST_ENV_NUMBER']` coming from?

Reply:
I don't remember! I just copy and pasted from the previous test lol